modules/removeBook.js

```diff
@@ -0,0 +1,26 @@
+import storageAvailable from './storageAvailable.js';
+import updateStorage from './updateStorage.js';
+import populateBook from './populateBook.js';
+
+const removeBook = (event) => {
+  let bookList = [];
+
+  if (storageAvailable('localStorage')) {
+    const { localStorage } = window;
+    bookList = JSON.parse(localStorage.getItem('bookList'));
+  }
+
+  const bookDiv = document.getElementsByClassName('book-list')[0];
+  const objectId = event.target.id;
+
+  // eslint-disable-next-line max-len
```

Comment:
- [ ] Please, consider removing all linter disables from your code. Linter should make its direct job - to help you find bugs for fix them on the beginning stage of your project 👍